test/compilable/test9029.d

```diff
@@ -0,0 +1,17 @@
+enum NameOf(alias S) = S.stringof;
```

Comment:
All test files should have a `TEST_OUTPUT` section (even if empty) to avoid that deprecations or warnings are silently ignored.

Reply:
fixed